lib/perfdata/elasticsearchwriter.ti

```diff
@@ -45,6 +45,10 @@ class ElasticsearchWriter : ConfigObject
 	[config] bool enable_ha {
 		default {{{ return false; }}}
 	};
+
+	[state, no_user_modify] int discovered_version {
+		default {{{ return 5; }}}
+	};
```

Comment:
Why 5?

Reply:
Well, the master code actually supports a whole range of ES versions: 5-7. But this is just a single number. I had to pick one. 🤷‍♂️